WORKSPACE

```diff
@@ -92,7 +92,7 @@ load("//dependencies/maven:artifacts.bzl", graknlabs_common_artifacts = "artifac
 ############################
 
 load("@graknlabs_dependencies//library/maven:rules.bzl", "maven")
-maven(graknlabs_common_artifacts)
+maven(graknlabs_common_artifacts + graknlabs_dependencies_tool_maven_artifacts)
```

Comment:
in the future i think if we try to be more careful when this `graknlabs_dependencies_tool_maven_artifacts` was originally added to the build file and think about why we need to import it, you'll remember to add it into this `maven` loading section too!